app/models.py

```diff
@@ -141,46 +153,24 @@ def __init__(self, **kwargs):
     # Column "id" is created automatically by SurrogatePK() from database.py
     uid = db.Column(db.String(64))  # DICOM Study UID (0020,000D)
     description = db.Column(db.String(100))  # DICOM Study Description (0008,1030)
+    # study_date = db.Column(db.String(64))  # DICOM Study date (0008,0020)
+    # study_time = db.Column(db.String(64))  # DICOM Study time (0008,0030)
```

Comment:
Any reason for commenting these out? Suggest retaining for now whilst we are developing.